data_steward/gcloud/gcs/__init__.py

```diff
@@ -151,5 +154,33 @@ def list_sub_prefixes(self, bucket_name: str, prefix: str) -> list:
         return list(pg_iterator)
 
     def _get_hpo_bucket_id(self, hpo_id: str) -> str:
-        # TODO reconsider how to map bucket name
-        return os.environ.get(f'BUCKET_NAME_{hpo_id.upper()}')
\ No newline at end of file
+        """[summary]
+
+        Args:
+            hpo_id (str): [description]
+
+        Raises:
+            ValueError: [description]
+            BucketDoesNotExistError: [description]
+
+        Returns:
+            str: [description]
+        """
+
+        project_id = os.environ.get('GOOGLE_CLOUD_PROJECT')
```

Comment:
app_identity should be used instead

Reply:
I was looking at this and started to wonder.  The `project_id` can get gathered directly through this client (`self.project`), and the `get_application_id()` in app_identity.py uses the project id as well.